core/admin.py

```diff
@@ -2,7 +2,7 @@
 from core.models import (
     PhonePrefix, WareHouse, Currency, Country,
     ContactUs, ProductType, News, FAQ,
-    CategoryFAQ, Discount, Tariff,
+    CategoryFAQ, Discount, Tariff,ForeignWarehouse
```

Comment:
Space after colon. Use autoformatter for smiplicity (black and isort)

Reply:
I remember promised about that. I just want to finish the remaining tasks before them